src/screens/OutboundStockDetails/index.tsx

```diff
@@ -99,7 +99,7 @@ class OutboundStockDetails extends React.Component<Props, State> {
     const searchTerm = input.toLowerCase();
     return this.state.shipment?.shipmentItems?.find(
       (item: ShipmentItems) =>
-        item.lotNumber?.toLowerCase().includes(searchTerm) ||
+        item.inventoryItem?.lotNumber?.toLowerCase().includes(searchTerm) ||
```

Comment:
Leave both `item.lotNumber` and `item.inventoryItem`, also check on the `OutboundStockList` (Packing list) if the same issue does not occur there too.

Reply:
@awalkowiak yeah, the same issue occured on `OutboundStockList` - below a commit fixing the issue there and also keeping the `item.lotNumber` on `OutboundStockDetails` which you mentioned